connector/connect/common/src/main/scala/org/apache/spark/sql/connect/client/SparkConnectClient.scala

```diff
@@ -252,8 +268,7 @@ private[sql] class SparkConnectClient(
       .setUserContext(userContext)
       .setSessionId(sessionId)
       .setClientType(userAgent)
-      .build()
-    bstub.releaseSession(request)
+    bstub.releaseSession(request.build())
```

Comment:
These two lines are equivalent right?

Reply:
Yes, this is just to make it uniform to everything else.